From 650e90a2b06e3ad51371ec5a83b7b5d96a740572 Mon Sep 17 00:00:00 2001
From: Yuta Saito <kateinoigakukun@gmail.com>
Date: Sat, 3 Aug 2024 07:25:16 +0000
Subject: [PATCH 1/2] [Runtime] Repair build of `getenv` mode in
 `EnvironmentVariables.cpp`

Follow-up fix to 4b3a197dc2eb0e1c83b847b250f448aa765ad4d5
---
 stdlib/public/runtime/EnvironmentVariables.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/stdlib/public/runtime/EnvironmentVariables.cpp b/stdlib/public/runtime/EnvironmentVariables.cpp
index 20491493e018b..ea5bfbd907e04 100644
--- a/stdlib/public/runtime/EnvironmentVariables.cpp
+++ b/stdlib/public/runtime/EnvironmentVariables.cpp
@@ -247,11 +247,11 @@ void swift::runtime::environment::initialize(void *context) {
   // everywhere.
 #define VARIABLE(name, type, defaultValue, help)                               \
   do {                                                                         \
-    const char name##_string = getenv(#name);                                  \
+    const char *name##_string = getenv(#name);                                 \
     if (name##_string)                                                         \
       name##_isSet_variable = true;                                            \
     name##_variable = parse_##type(#name, name##_string, defaultValue);        \
-  } while (0)
+  } while (0);
 #include "EnvironmentVariables.def"
 
   // Print help if requested.

